parastell/magnet_coils.py

```diff
@@ -269,6 +269,29 @@ def export_step(self, step_filename="magnet_set", export_dir=""):
         )
         cq.exporters.export(coil_set, str(export_path))
 
+    def export_stl(self, stl_filename="magnet_set", export_dir=""):
```

Comment:
Similar comment as above about extending the previous method rather than copy/pasting to a new method.

Reply:
I see. I did this way because It was usefull for me to get these two format at the same time, but for sure it can be done extending the method instead of duplicating it. I'l have a deeper look on it

Reply:
You could still get both in the same run, by just calling the method twice with different arguments.